src/dex-ui/pages/dao/utils.ts

```diff
@@ -98,3 +106,31 @@ export function getProposalSteps(
   }
   return { steps, activeStep };
 }
+
+export function getProposalData(proposal: Proposal): string {
+  if (proposal.type === ProposalType.AddNewMember) {
+    const { owner, _threshold } = proposal.data as ProposalDataAddMember;
+    console.log("[ANS] owner", proposal.data, owner, proposal);
+    const ownerAddress = owner ? solidityAddressToTokenIdString(owner) : "";
+    return `Proposed to add new member ${ownerAddress} and change threshold to ${_threshold}`;
+  }
+  if (proposal.type === ProposalType.RemoveMember) {
+    const { owner, _threshold } = proposal.data as ProposalDataDeleteMember;
+    const ownerAddress = solidityAddressToTokenIdString(owner);
```

Comment:
A call to the `solidityAddressToTokenIdString` function is crashing the app when a Remove Member proposal is created:

<img width="958" alt="Screen Shot 2023-08-02 at 10 14 10 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/6c40babe-45f9-4234-b5ec-0c2d09f79d95">